runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerCommand.java

```diff
@@ -36,10 +36,16 @@
 
 /** A docker command wrapper. Simplifies communications with the Docker daemon. */
 class DockerCommand {
+
+  private static final String DEFAULT_DOCKER_COMMAND = "docker";
   // TODO: Should we require 64-character container ids? Docker technically allows abbreviated ids,
   // but we _should_ always capture full ids.
   private static final Pattern CONTAINER_ID_PATTERN = Pattern.compile("\\p{XDigit}{64}");
 
+  public static DockerCommand getDefault() {
```

Comment:
Just a small note: there's not much point making this public since the class is package-private.

Reply:
That's true; but if it becomes public in the future, this seems like a sane constructor to expose.